src/main/resources/application.yaml

```diff
@@ -0,0 +1,68 @@
+# Server details
+server:
+  port: 8080
+  shutdown: graceful
+  servlet:
+    context-path: /api
+
+
+# Actuator endpoint configurations
+info:
+  app:
+    version: 0.0.1
+    release: '0_20210610'
+    name: ${spring.application.name}
+    servlet:
+      context-path: /NotificationRelayApp
```

Comment:
You have just completely copy pasted code from my example. Is you application name `NotificationRelayApp`?

Infact in all you class implementations you have not even made any attempts to change the names according to you application.

Reply:
I accept that most of the code is same from your example project, I apologize for that but I thought that is boilerplate code. I definitely should have changed some things. Sorry for that.